services/provider/server/storageclassrequest.go

```diff
@@ -70,7 +70,7 @@ func (s *storageClassRequestManager) Create(
 			Namespace: s.namespace,
 			Labels: map[string]string{
 				controllers.ConsumerUUIDLabel: consumerUUID,
-				storageClassRequestName:       storageClassRequestName,
+				storageClassRequestName:       storageClassRequestNameLabel,
```

Comment:
I think it should be vice versa @Madhu-1 WDYT?

Reply:
Yes it should always be key:value  key is the const identifier and value should be the request name

Reply:
@subhamkrai Can you pls change it as Madhu as confirmed it?

Reply:
right 😅